### PR TITLE
Sermant Community Interest Discussion Meeting Minute Book

### LINKED ISSUE #331: Update Sermant Sermant Community Interest Discussion Meeting Minutes
### What would you like to be added?

Update Sermant Sermant Community Interest Discussion Meeting Minutes

### Why is this needed?

Sync the latest Sermant Community Interest Discussions

### PR DESCRIPTION
**Pull request by daizhenyu**
What type of PR is this?

/feat

What this PR does / why we need it?

Update the Sermant Community Interest Discussion Meeting Minutes.

Which issue(s) this PR fixes:

Fixes #331

### CODE REVIEW

File: docs/zh/document/community/sig.md [hunk 1]
```diff
@@ -1,14 +1,5 @@
...
-**会议议题**：
-1. 研讨Issue 1445：[Sermant 能够支持 Prometheus 监控](https://github.com/sermant-io/Sermant/issues/1445)
-
-**会议链接**：[2024/03/08 下午 15:00--15:45 UTC+8](https://meeting.tencent.com/dm/oRvUhbxwBpMx)
-
-**会议纪要**：
-1. Sermant社区接纳Issue1445的需求，支持Prometheus监控Sermant框架、插件以及宿主微服务的指标信息。
-2. Sermant实现监控能力接入首先基于Prometheus的原生SDK能力实现，根据后续社区需求，实现灵活且可扩展的自定义Metric接口，并与多种监控系统兼容。
+欢迎关注Sermant社区的兴趣讨论板块，[Sermant社区兴趣讨论会会议纪要](https://docs.google.com/document/d/11Ln1MzOil0JjkM8SEeaJPukumIFyM_eG_u9WMQyke4c/edit)
+发布和记录了兴趣讨论会的会议信息。
```
Comment (lilai23) [inline docs/zh/document/community/sig.md]: 记录就可以了，发布感觉没必要

File: docs/en/document/community/sig.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# Interest Discussions
+Welcome to follow the Sermant Community Interest Discussion section. The [Sermant Community Interest Discussion 
+Meeting Minute Book](https://docs.google.com/document/d/11Ln1MzOil0JjkM8SEeaJPukumIFyM_eG_u9WMQyke4c/edit) publish 
+and record the information from the interest discussion meetings.
```
Comment (lilai23) [inline docs/en/document/community/sig.md]: records

File: docs/zh/document/community/sig.md (hunk 1 above)
Comment (daizhenyu) [reply docs/zh/document/community/sig.md]: doen

File: docs/en/document/community/sig.md (hunk 2 above)
Comment (daizhenyu) [reply docs/en/document/community/sig.md]: done

**Review by lilai23 — APPROVED** [review]

**Pull request merged**